docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
     volumes:
       - ./src:/usr/src/nijobs-be/src
       - ./test:/usr/src/nijobs-be/test
+      - ./coverage:/usr/src/app/coverage
```

Comment:
I'm curious, any reason to use the `app/` folder instead of `nijobs-be` like the other volumes?

Reply:
The [dockerfile](https://github.com/NIAEFEUP/nijobs-be/blob/c272e3bbbda9937b4ec7c2c4d8ac6bbb811684c7/Dockerfile-test#L4) is using the `app` folder as the workdir, not sure why the other volumes use `nijobs-be` :thinking:

Reply:
You're right :upside_down_face: Seems like those volumes aren't even used... Can you fix this? The `web-dev` service is correct, this is the incorrect one